.github/workflows/issue-management-update-harvester-projects.yaml

```diff
@@ -65,6 +54,17 @@ jobs:
         project-url: ${{ env.HARVESTER_SPRINT_PROJECT_URL }}
         github-token: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
 
+    - name: Set Milestone to Issue
+      if: ${{
+        fromJSON(steps.is-harvester-member.outputs.teams)[0] != null  &&
+        github.event.issue.milestone == null
+        }}
+      env:
+        GH_TOKEN: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
+      run: |
+        ISSUE_NUMBER="${{ github.event.issue.number }}"
+        gh issue edit "$ISSUE_NUMBER" --milestone "Planning"
+        
```

Comment:
We should add it Planning milestone after we add it into Harvester sprint, not before.